### PR TITLE
Api basic features.

### PR DESCRIPTION
**Pull request by gabrielyea**
# Submit and get data from API! :shipit: 
- Submit your name and score!
- Score is sorted!
- Submit button disables while waiting for promise resolve!
- Made the score table 'blink' so the user can tell when it refreshed.

[Check it out!](https://gabrielyea.github.io/leaderboard/)

### CODE REVIEW

File: src/scripts/components/utils/localStorage.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+export default class LocalStorage {
+  saveData = ({ data }) => {
+    window.localStorage.setItem('data', JSON.stringify(data));
+  };
+
+  /**
+   * Gets all loaded user data from previous session.
```
Comment (carlos-ssh) [inline src/scripts/components/utils/localStorage.js]: [OPTIONAL]
- Keep your project professional and remove unused comments in the code.

File: src/scripts/components/leaderboard/leaderboard-interface.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import leaderboardActions from './leaderboard-actions.js';
+
+class LeaderboardInterface {
+  form = document.querySelector('form');
+
+  refresh = document.querySelector('.refresh-btn');
```
Comment (carlos-ssh) [inline src/scripts/components/leaderboard/leaderboard-interface.js:6]: **Kindly make a few changes:**

- Implement the "Refresh" button (receiving data from the API and parsing the JSON). 
(Your refresh button isn't working).

- Fix the error issue that appears in the chrome console:
``
Uncaught (in promise) TypeError: Cannot read property 'inline' of undefined
``

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @gabrielyea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scripts/components/leaderboard/leaderboard-interface.js (hunk 2 above)
Comment (gabrielyea) [reply src/scripts/components/leaderboard/leaderboard-interface.js:6]: hello @carlos-ssh! the refresh button is working, but there is nothing to refresh when you submit data. (the scores already are on the leaderboard). But how is it supposed to work? are we supposed to keep one static id for our game? or are we supposed to create a new game every time we refresh the page?

Comment (gabrielyea) [reply src/scripts/components/leaderboard/leaderboard-interface.js:6]: Was not able to reproduce the error, I tried in all browsers.

**Comment by Davidosky007** [comment]
## Hi @gabrielyea 👋, 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏
# [optional]
* Please note that for javascript projects the right linter to use is [Javascript linters](https://github.com/microverseinc/linters-config/tree/master/javascript) and not [HTML + CSS + JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) 
* The HTML+CSS+JS is made for the Portfolio project which needs Javascript for interactions as a result the linters was made for easy use 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**